app/src/main/java/com/a0xffffffff/dinesum/MainActivity.java

```diff
@@ -229,25 +229,12 @@ public void onSubmitButtonPressed(String TAG, Request request) {
 
     @Override
     public void onUpdateRequestState(String TAG, Request request) {
-        if (TAG.equals(RequestFeedFragment.TAG)) {

```

Comment:
later we can probably combine all of the "if (TAG.equals(...))" checks into one "if" statement